pytest_html/resources/main.js

```diff
@@ -62,14 +62,23 @@ function hide_extras(colresult_elem) {
     expandcollapse.classList.add("expander");
 }
 
+function add_filter_boxes() {
+    var checkboxes = document.getElementsByName('filter_checkbox');
+    for (var i = 0; i < checkboxes.length; i++)
+      checkboxes[i].hidden = false;
+
+    var checkbox_text = document.getElementById('filter_checkbox_text');
+    checkbox_text.hidden = false;
```

Comment:
As mentioned above, by using a common class name we could then do `document.getElementsByClassName('filter')`. This function could then be renamed to `show_filters` and used for toggling visibility of all filter related elements that might be added in the future.